Docs/UPDATING.md

```diff
@@ -0,0 +1,27 @@
+# How to support new version of Xcode
+
+When Xcode updates, it might change the format of its messages to XCBBuildService.
+
+This project makes thorough use of [swift-logging](https://github.com/apple/swift-log), so to see what's wrong, look in the logs!
+
+You can start recording logs by executing `write_shim.sh`. You'll need your built `BazelXCBBuildService` and the path to where Xcode is expecting `XCBBuildService`. You also pass in the Xcode version, so your logs will be split based on the Xcode version.
+
+The `XCBBuildService` lives here: `$(XCODE_DIR)/Contents/SharedFrameworks/XCBuild.framework/PlugIns/XCBBuildService.bundle/Contents/MacOS/XCBBuildService`
+
+`write_shim.sh` will create a fake `XCBBuildService` that simply calls the `BazelXCBBuildService` while redirecting `STDERR` to a log file in `/tmp/Bazel-trace/`.
+
+### Prepping for a new version of Xcode:
+* Create a new module inside `/Sources` following the established pattern. (e.g. `XCBProtocol_14_0`)
```

Comment:
We should add some notes here about how, unless there are changes to files, they exist just a symlinks to previous versions of the same file. My personal workflow is to create the new directory, then go through and convert any non-symlink files into symlinks pointing at the previous directory's version of the files.

Reply:
I'll add this 👍